lib/CLI.js

```diff
@@ -2369,7 +2369,7 @@ function checkDeprecates(conf){
  */
 function prepareAppName(conf){
   if (!conf.name){
-    conf.name = p.basename(conf.script);
+    conf.name = conf.script !== undefined ? p.basename(conf.script) : 'undefined';
```

Comment:
can't this be fixed on a higher level? Also, having `conf.name = 'undefined'` can't be correct.